Client/Frontend/Browser/Authenticator.swift

```diff
@@ -7,7 +7,7 @@ import Shared
 import Storage
 import Deferred
 
-private let CancelButtonTitle = NSLocalizedString("Cancel", comment: "Authentication prompt cancel button")
+private let CancelButtonTitle = NSLocalizedString("Cancel", comment: "Label for Cancel button")
```

Comment:
If we want to have the exact same string and comment everywhere we have "Cancel", we might as well just use `UIConstants.CancelString` everywhere we need it instead of defining new `NSLocalizedString`s. I can file a new bug for this if you don't want to make that change here.